docs/index.md

```diff
@@ -1,9 +1,27 @@
 # AMD ROCm™ Platform Documentation
+AMD ROCm Platform - Powering Your GPU Computational Needs
+:::::{grid} 1 1 3 3
+:gutter: 1
 
-Welcome to AMD ROCm™ Platform's documentation!
+::::{grid-item}
+:::{dropdown} [Deploy ROCm](deploy)
+ 
+- {doc}`/deploy/linux/index`
+- {doc}`/deploy/quick_start_windows`
+- {doc}`/deploy/docker`
 
-:::::{grid} 1 1 2 2
-:gutter: 1
+:::
+::::
+
+::::{grid-item}
+:::{dropdown} [What is ROCm?](rocm)
```

Comment:
It might be more intuitive to have this directly on the page (as in not in a grid), or as the first grid item (i.e.: before `Deploy ROCm`) if there are concerns about saving space.

Reply:
Space isn't that much of a concern but you are right, it could be the first before deploy. I was thinking differently. I wanted What is ROCm to eventually be a gateway to how we do ROCm releases, how we version, explanations on userspace, kernel space, etc.